### PR TITLE
swap default model to auto for free provider

### PR DESCRIPTION
**Pull request by tarasglek**
@humphd or someone else, can you help me fix the blank circle?

image

### CODE REVIEW

**Comment by kliu57** [comment]
@tarasglek I'll look into this

**Comment by kliu57** [comment]
> @humphd or someone else, can you help me fix the blank circle?

@tarasglek I have fixed the blank logo problem in this PR -> https://github.com/tarasglek/chatcraft.org/pull/558

File: .vscode/settings.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.formatOnSave": true,
   "editor.codeActionsOnSave": {
-    "source.fixAll": true
```
Comment (Rachit1313) [inline .vscode/settings.json:8]: I believe you forgot to take out this change.

**Review by Rachit1313 — CHANGES_REQUESTED** [review]

**Comment by tarasglek** [comment]
@kliu57 after i merged your change into main, it stopped working

**Comment by tarasglek** [comment]
Can you try again(just land on top of this pr please)

**Comment by kliu57** [comment]
@tarasglek Sorry taras, this is my first time trying to remote into someone else's branch. I think I typed some command which merged it into main after I added my fix.

**Comment by kliu57** [comment]
I was following a step by step tutorial on how to work on another person's branch, and I think that tutorial was working with main, which is I ended up pushing to main. My fix is there along with your other commits.

image

Again sorry about that

**Comment by humphd** [comment]
@kliu57 so is this OK now or do you need help fixing?

**Pull request merged**